prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -138,13 +140,25 @@ class FakeDataService {
                         )
                 }
                 element.table?.startsWith("LIVD-Supplemental") == true -> {
-                    if (element.tableColumn == null) return ""
+                    if (element.tableColumn == null)
+                        return ""
                     element.default ?: ""
                 }
                 element.table == "fips-county" -> {
                     when {
                         element.nameContains("state") -> context.state
                         element.nameContains("county") -> context.county
+                        (element.default == null) -> "${element.default}"
+                        else -> TODO("Add this column in a table")
+                    }
+                }
+                element.table == zipCodeData -> {
+                    when {
+                        element.nameContains("state") -> context.state
+                        element.nameContains("county") -> context.county
+                        element.nameContains("zip") -> context.zipCode
+                        element.nameContains("city") -> context.city
+                        (element.default == null) -> "${element.default}"
```

Comment:
Change this to return "" instead.  No need to have element.default inside

Reply:
Done.